src/main/java/ua/hillel/moneymanager/repository/CategoryRepository.java

```diff
@@ -0,0 +1,20 @@
+package ua.hillel.moneymanager.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import ua.hillel.moneymanager.entity.Category;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface CategoryRepository extends JpaRepository<Category, Long> {
+
+    @Query(value = "select * from categories where categories.user_id = ?1",
+            nativeQuery = true)
+    List<Category> findAllByUserId(Long UserId);
+
+    @Query(value = "update categories set name = ?2 where id = ?1",
+            nativeQuery = true)
+    Category updateById(Long id, String name);
```

Comment:
Why do you need to update only a name of a category? How about the category's balance or other fields?